### PR TITLE
Fix example link and some typo in Readme

### PR DESCRIPTION
**Pull request by khafatech**
The examples link was wrong, and I fixed minor typo & punctuation in the sentence about `SkimItemReader`

### CODE REVIEW

**Comment by LoricAndre** [comment]
Could you also change the old repository url to the new org please ?

**Comment by khafatech** [comment]
@LoricAndre Good catch! I updated the url to this repository

File: README.md [hunk 1]
```diff
@@ -396,10 +396,10 @@ Trait `SkimItem` is provided to customize how a line could be displayed,
 compared and previewed. It is implemented by default for `AsRef<str>`
 
 Plus, `SkimItemReader` is a helper to convert a `BufRead` into
-`SkimItemReceiver` (we can easily turn a `File` for `String` into `BufRead`).
-So that you could deal with strings or files easily.
+`SkimItemReceiver` (we can easily turn a `File` or `String` into `BufRead`),
+so that you could deal with strings or files easily.
 
-Check more examples under [examples/](https://github.com/lotabout/skim/tree/master/examples) directory.
+Check out more examples under the [examples/](https://github.com/lotabout/skim/tree/master/skim/examples) directory.
```
Comment (LoricAndre) [inline README.md]: No `the` here I think, under `folder` sounds better to my ear at least

**Comment by LoricAndre** [comment]
Have you proofread the rest of the README ? I'd like to avoid merging a PR this small if we can

File: README.md (hunk 1 above)
Comment (khafatech) [reply README.md]: Without `the`, we would have to remove "directory." I don't know how to explain it, it just sounds more right to me.

I feel like we can have either of these sound good. 
* "Check out more examples under the [examples/](link) directory."
* "Check out more examples under examples."

**Comment by khafatech** [comment]
> Have you proofread the rest of the README ? I'd like to avoid merging a PR this small if we can

I haven't. Good idea, I'll proof read the rest of the readme and add a new commit

File: README.md (hunk 1 above)
Comment (LoricAndre) [reply README.md]: I missed "directory", you're right

**Review by LoricAndre — APPROVED** [review]

**Pull request merged**